src/components/header/header.module.scss

```diff
@@ -4,6 +4,8 @@
     --headerTopBarHeight: 50px;
     --headerNavigationHeight: 54px;
     --headerHeight: calc(var(--headerTopBarHeight) + var(--headerNavigationHeight));
+
+    --actionsWidth: 500px;
```

Comment:
isn't this too big on smaller screens?
We hide site menu if viewport is less than 1024.. on such viewport I don't think that are 500px to the right of the site menu.

Reply:
I mean here 
<img width="1026" alt="image" src="https://github.com/user-attachments/assets/13ed8d89-aea6-4ef6-8508-5262c37d9c4d">

Reply:
refactored layout